tests/e2e/test_login-signup.py

```diff
@@ -0,0 +1,27 @@
+from utils import heavily_populate_db, populate_db, clear_db
+from app import repository_singleton
+from src.models import AppUser
+
+def test_client():
+    clear_db()
+    populate_db()
+
+def test_signup(test_client):
+    response = test_client.post('/process_form', data={'username':'new_user', 'hashed_password':'password', 'action':'Sign Up'})
+    assert response.status_code == 200
```

Comment:
should be 302, since signing up redirects you (to the same page, but it's still a redirect). will only return 200 if "follow_redirects=true" is in the data dictionary in line 10